types/ko.plus/index.d.ts

```diff
@@ -156,3 +159,9 @@ declare namespace KoPlus {
 
     //#endregion
 }
+
+declare var ko: KnockoutStatic;
+
+declare module "ko.plus" {
```

Comment:
Can't you just say `export = ko`? I don't understand why the ambient module declaration is needed.

Reply:
Without the declaration of the ambient module module loaders won't see the library, e.g.
```
import * as koplus from 'ko.plus';
```